### PR TITLE
Fix issue #31

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@tejohnso please review. cheers

### CODE REVIEW

File: rise-financial-es6.js [hunk 1]
```diff
@@ -294,7 +294,8 @@
         event: "ready"
       };
 
-      if ( !this._firebaseApp ) {
+      // ensure firebase app has not been initialized already
+      if ( !this._firebaseApp && !firebase.apps.length ) {
```
Comment (tejohnso) [inline rise-financial-es6.js]: @rodrigopavezi If firebase has allready been initialized and the second test fails then this._firebaseApp isn't set. Shouldn't it be set to the existing instance that caused the second test to fail?

**Review by tejohnso — CHANGES_REQUESTED** [review]

**Comment by rodrigopavezi** [comment]
@tejohnso good catch. I have changed that. I have also fixed another issue with the JSONP callback so it will be unique per instance of the component. Please review. cheers

**Review by tejohnso — APPROVED** [review]

**Review by stulees — CHANGES_REQUESTED** [review]
@rodrigopavezi This needs a version bump and rebuild, thanks.

**Comment by rodrigopavezi** [comment]
@stulees version bumped, please review. cheers

**Review by stulees — APPROVED** [review]

**Comment by stulees** [comment]
@rodrigopavezi 👍

**Pull request merged**